package.json

```diff
@@ -28,7 +28,8 @@
         "jsdom": "~3.1.0",
         "bluebird": "~2.9.1",
         "lodash": "~2.4",
-        "qs": "2.3.3"
+        "qs": "2.3.3",
```

Comment:
yes to match the implementation that our version of express is using. the reasoning is express parses the query string a certain way. in order to match it on the client side, we should use the exact same version and parsing strategy